precommit/src/main/shell/plugins.d/github.sh

```diff
@@ -1143,6 +1179,7 @@ function github_finalreport
   # - failure w/log
   # - success w/warning log
 
+  url=$(get_artifact_url)
```

Comment:
Minor nit: why move population of the `url` value? Does the `artifact_url` change from the beginning of this function call to down here?

Reply:
It doesn't and this change was a result of some of the development work that happened previously.